Makefile

```diff
@@ -1,22 +1,22 @@
 GO111MODULE := on
 export GO111MODULE
 
-all: check test race
+all: build lint test
```

Comment:
nice that now `test` runs `race` by default! Would be nice to add a `clean` option, which is trivial but idiomatically aligns with the rest of the targets

Reply:
Done